src/core/gpu_types.h

```diff
@@ -198,7 +198,7 @@ union GPUTexturePaletteReg
   BitField<u16, u16, 6, 10> y;
```

Comment:
Does changing this from `BitField<u16, u16, 6, 10> y` to `BitField<u16, u16, 6, 9> y` also fix it? The code should be the same as the mask/shift.